t/producev.t

```diff
@@ -0,0 +1,81 @@
+use strict;
+use warnings;
+use Test::More;
+
+my $message_max_bytes = 1000;
+
+BEGIN { use_ok "Kafka::Librd" }
+my $krd = Kafka::Librd->new(
+    Kafka::Librd::RD_KAFKA_CONSUMER,
```

Comment:
RD_KAFKA_CONSUMER? do we have RD_KAFKA_PRODUCER we can use?

Reply:
Yeah, we have that. From `rdkafka.h`:

```
typedef enum rd_kafka_type_t {
	RD_KAFKA_PRODUCER, /**< Producer client */
	RD_KAFKA_CONSUMER  /**< Consumer client */
} rd_kafka_type_t;
```

So it would be correct to use `RD_KAFKA_PRODUCER` here. Updated! :smile: